packages/ui-docs-client/src/Header/index.js

```diff
@@ -29,13 +29,15 @@ import { Link } from '@instructure/ui-link'
 import { InlineSVG } from '@instructure/ui-svg-images'
 import { Text } from '@instructure/ui-text'
 import { View } from '@instructure/ui-view'
-import { themeable } from '@instructure/ui-themeable'
 
 import { Heading } from '../Heading'
 
-import theme from './theme'
+import { withStyle } from '@instructure/emotion'
 
-@themeable(theme)
+@withStyle(
+  (componentTheme, props, state) => ({}),
+  (theme, themeOverride = {}) => ({})
+)
```

Comment:
Since we don't use any styling here, the withStyle decorator can be deleted altogether.